Describe empty topics in DataViz and Sortable (#954)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxSortable/1 Configuration/onReorder.md

Model data. Available only if you use Knockout.

##### field(e.promise): Promise<void>

> Row dragging is finished once this Promise is resolved.

Reviewer: ```suggestion
A Promise that is resolved when the item is dropped.
```
